src/js/components/awardv2/idv/federalAccounts/FederalAccountsSummary.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import FederalAccountsTable from './FederalAccountsTable';
+import { formatMoneyWithPrecision } from '../../../../helpers/moneyFormatter';
+import { Table } from "../../../sharedComponents/icons/Icons";
+
+const propTypes = {
+    totalTransactionObligatedAmount: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    awardingAgencyCount: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    federalAccountCount: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    jumpToFederalAccountsHistory: PropTypes.func,
+    inFlight: PropTypes.bool,
+    error: PropTypes.bool,
+    page: PropTypes.number,
+    limit: PropTypes.number,
+    sort: PropTypes.string,
+    order: PropTypes.string,
+    total: PropTypes.number,
+    federalAccounts: PropTypes.array,
+    changePage: PropTypes.func,
+    updateSort: PropTypes.func
+};
+
+const FederalAccountsSummary = ({
+    totalTransactionObligatedAmount,
+    awardingAgencyCount,
+    federalAccountCount,
+    jumpToFederalAccountsHistory,
+    inFlight,
+    error,
+    page,
+    limit,
+    sort,
+    order,
+    total,
+    federalAccounts,
+    changePage,
+    updateSort
```

Comment:
These are a lot of props for this component. Makes it hard to read at a glance. I would recommend moving the FederalAccountsTable outside of this component, and putting it into the parent component. 

So then "AwardMetaDataContainer" would render two seaparate components.

Reply:
The tree map PR creates a FederalAccountsSection class that handles rendering the Toggle Buttons, Error/Loading/No Results Messages, TreeMap, and Table

Reply:
TreeMap PR has been updated from a FederalAccountsSummary class to component

Reply:
@MaxwellKendall